tests/CommandLineTestRunnerTest.cpp

```diff
@@ -94,3 +95,25 @@ TEST(CommandLineTestRunner, NoPluginsAreInstalledAtTheEndOfARunWhenTheArgumentsA
 
     LONGS_EQUAL(0, registry.countPlugins());
 }
+
+struct TestOutputCheckingCommandLineTestRunner : public CommandLineTestRunner
+{
+    TestOutputCheckingCommandLineTestRunner(int ac, const char** av, TestOutput* output, TestRegistry* registry) :
+        CommandLineTestRunner::CommandLineTestRunner(ac, av, output, registry)
+    {
+    }
+
+    bool hasJUnitTestOutput(void)
+    {
+        return NULL != dynamic_cast<JUnitTestOutput*>(output_);
```

Comment:
Could you explain how? My point is to assert that output_ has been instantiated to a JUnitTestOuput rather than StringBufferTestOutput. How could the CommandLineArguments class help with that?